### PR TITLE
Added README.md file.

### PR DESCRIPTION
**Pull request by Rohan-cod**

### CODE REVIEW

**Comment by zhuyifei1999** [comment]
-1. This completely doesn't explain what the project does, only explains what Toolforge and MediaWiki are. A README should explain about _this_ project, not _some other_ projects. The "website" isn't even the public endpoint, IRC is.

**Comment by Rohan-cod** [comment]
Actually I wasn't able to find much information about media-dubiety. 
If you can provide me with a link to more information about it I will add important points to the Readme. It would really help me. I will also change the link to point to the correct url.

**Comment by zhuyifei1999** [comment]
T129845 is non-public and relates to [these discussions](https://commons.wikimedia.org/w/index.php?sort=relevance&search=%22Influx+of+files+with+embedded+data+%28CSD%23F9%29%22&title=Special:Search&profile=advanced&fulltext=1&advancedSearch-current=%7B%7D&ns4=1), although it has more of the embeddeddata project than this one.

This project is an IRC bot in channels defined in `~tools.media-dubiety/channels.json`. It sends messages such as:
```
<media-dubiety> https://tinyurl.com/CentralAuth/REDACTED (3 edits) uploaded https://commons.wikimedia.org/wiki/File:REDACTED (56.7MiB, 5 min)
<media-dubiety> REDACTED blocks User:REDACTED on simplewiki for: Reciprocal block: Violating the "one-strike" rule
```
The purpose of this project used to be T129845, but right now it's more like "if abuse happens again we would know".

**Comment by Rohan-cod** [comment]
Thanks for the help @zhuyifei1999, I will update the Readme file with the correct information.
Can you tell me which url I am supposed to add.

**Comment by zhuyifei1999** [comment]
> Can you tell me which url I am supposed to add.

I'd rather not have a URL; it doesn't have a "webservice" in Toolforge terms and I don't see a reason to document detail abuse-prevention mechanisms publicly.

**Comment by Rohan-cod** [comment]
I am adding a Readme so that whenever a new user visits the repository he gets some information regarding the project like what it is about, what is the history and other stuff.

**Comment by Rohan-cod** [comment]
@zhuyifei1999, I have updated the README, please review.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,10 @@
 # Media Dubiety
-Rewrite of Dispenser's NewbieAudioVideo IRC bot for T129845
+**Media Dubiety** is an IRC bot in channels defined in `~tools.media-dubiety/channels.json`. It sends messages such as:
```
Comment (zhuyifei1999) [inline README.md]: `~tools.media-dubiety/channels.json` applies for this instance of the running bot.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,10 @@
...
+* `<media-dubiety> https://tinyurl.com/CentralAuth/REDACTED (3 edits) uploaded https://commons.wikimedia.org/wiki/File:REDACTED (56.7MiB, 5 min)`
+* `<media-dubiety> REDACTED blocks User:REDACTED on simplewiki for: Reciprocal block: Violating the "one-strike" rule`
 
-Website: [Media Dubiety](https://tools.wmflabs.org/admin/tool/media-dubiety)
+## T129845
```
Comment (zhuyifei1999) [inline README.md]: No need to advertise it.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,10 @@
...
-
-To learn more about toolforge [click here](https://wikitech.wikimedia.org/wiki/Portal:Toolforge).
-
-## Mediawiki
-MediaWiki is a free server-based software, licensed under the GNU General Public License (GPL). 
-It's designed to run on a large server farm for a website that gets millions of hits per day.
-
-To learn more about Mediawiki [click here](https://www.mediawiki.org/wiki/Manual:What_is_MediaWiki%3F).
+## Purpose
+The purpose of this project used to be **T129845**, but right now it's more like "*if abuse happens again we would know*".
```
Comment (zhuyifei1999) [inline README.md]: No need to advertise it.

**Review by zhuyifei1999 — CHANGES_REQUESTED** [review]
Look, I really don't think it's a good idea to advertise anti-abuse mechanisms on the "front page". 

Example messages? Sure. People knows what this does.

Anything about T129845? No thanks. That was for your information only.

**Comment by Rohan-cod** [comment]
> Look, I really don't think it's a good idea to advertise anti-abuse mechanisms on the "front page".
> 
> Example messages? Sure. People knows what this does.
> 
> Anything about T129845? No thanks. That was for your information only.

Done.

**Comment by zhuyifei1999** [comment]
Gonna add this additional patch and squash the commits:

```
From 8355833be40b40ba96b3a9f417b2648999662f71 Mon Sep 17 00:00:00 2001
From: YiFei Zhu <zhuyifei1999@gmail.com>
Date: Tue, 10 Mar 2020 03:23:25 -0500
Subject: [PATCH] README.md: Clarify channels def & use foobar for less
 obnoxious redaction

---
 README.md | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/README.md b/README.md
index 1a75d6c..eaf65ad 100644
--- a/README.md
+++ b/README.md
@@ -1,4 +1,5 @@
 # Media Dubiety
-**Media Dubiety** is an IRC bot in channels defined in `~tools.media-dubiety/channels.json`(When the bot is running). It sends messages such as:
-* `<media-dubiety> https://tinyurl.com/CentralAuth/REDACTED (3 edits) uploaded https://commons.wikimedia.org/wiki/File:REDACTED (56.7MiB, 5 min)`
-* `<media-dubiety> REDACTED blocks User:REDACTED on simplewiki for: Reciprocal block: Violating the "one-strike" rule`
+**Media Dubiety** is an IRC bot in channels defined in `~/channels.json`. It sends messages such as:
+
+* `<media-dubiety> https://tinyurl.com/CentralAuth/Foo (3 edits) uploaded https://commons.wikimedia.org/wiki/File:Bar.webm (56.7MiB, 5 min)`
+* `<media-dubiety> Baz blocks User:Foo on commonswiki for: Disposable account for uploading copyrighted content`
-- 
2.25.1
```

**Comment by Rohan-cod** [comment]
Do I have to send a new pr now?

**Comment by zhuyifei1999** [comment]
No, it has been "squash" merged.

**Comment by Rohan-cod** [comment]
Okay 😀.

**Pull request closed without merging**